assignments/array-methods.js

```diff
@@ -56,28 +56,71 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function(array){
+    fullName.push(array.first_name + ' ' + array.last_name)
```

Comment:
```suggestion
    fullName.push(`${array.first_name} ${array.last_name}`)
```
The way you have it is also fine, I'm just sharing this alternate syntax